furious/async.py

```diff
@@ -88,7 +88,7 @@ def run_me(*args, **kwargs):
 
 
 ASYNC_DEFAULT_QUEUE = 'default'
-ASYNC_ENDPOINT = '/_ah/queue/async'
+ASYNC_ENDPOINT = '/_queue/async'
```

Comment:
It's OSS, so many "could" be. Making it a config option is probably a good idea but doesn't seem like a requirement atm.